app/scripts/metamask-controller.js

```diff
@@ -812,6 +815,8 @@ export default class MetamaskController extends EventEmitter {
           this.claimsController.fetchClaimsConfigurations().catch((err) => {
             log.error('Error fetching claims configurations', err);
           });
+          // shield subscribers have to turn on metametrics
+          this.metaMetricsController.setParticipateInMetaMetrics(true);
```

Comment:
can u also move other settings we're turning on to here from shield settings page